io/include/pcl/compression/impl/organized_pointcloud_compression.hpp

```diff
@@ -292,7 +292,7 @@ namespace pcl
       std::vector<std::uint8_t> compressedColor;
 
       std::uint32_t compressedDisparitySize;
-      std::uint32_t compressedColorSize;
+      std::uint32_t compressedColorSize = 0;
```

Comment:
Not needed, since we're checking both the value after reading and the success of reading.
```suggestion
      std::uint32_t compressedColorSize;
```